kbase-extension/scss/partials/_custom.scss

```diff
@@ -393,3 +393,21 @@ div.code_cell {
     content: '\f054 ';
   }
 }
+
+/* up/down caret expander */
```

Comment:
This is very similar to the Bootstrap collapse styling; unfortunately there is no automagic show/hide behaviour to go with it.